drivers/pds/lib/utils.go

```diff
@@ -229,8 +234,28 @@ func GetAndExpectBoolEnvVar(varName string) (bool, error) {
 	return varBoolValue, err
 }
 
+func GetDeploymentTargetID(clusterID, tenantID string) (string, error) {
+	log.Info("Get the Target cluster details")
+	targetClusters, err := components.DeploymentTarget.ListDeploymentTargetsBelongsToTenant(tenantID)
+	if err != nil {
+		log.Errorf("Error while listing deployments %v", err)
+		return "", err
+	}
+	if targetClusters == nil {
+		log.Fatalf("Target cluster passed is not available to the account/tenant %v", err)
+	}
+	for i := 0; i < len(targetClusters); i++ {
+		if targetClusters[i].GetClusterId() == clusterID {
+			deploymentTargetID = targetClusters[i].GetId()
+			log.Infof("deploymentTargetID %v", deploymentTargetID)
+			log.Infof("Cluster ID: %v, Name: %v,Status: %v", targetClusters[i].GetClusterId(), targetClusters[i].GetName(), targetClusters[i].GetStatus())
+		}
+	}
+	return deploymentTargetID, nil
+}
+
 // SetupPDSTest returns few params required to run the test
-func SetupPDSTest(ControlPlaneURL, ClusterType, AccountName string) (string, string, string, string, string, error) {
+func SetupPDSTest(ControlPlaneURL, ClusterType, AccountName, TenantName, ProjectName string) (string, string, string, string, string, error) {
```

Comment:
this seems more like to define in common.go file

Reply:
yes you are right, I tried moving to common.go file but its giving some issues related to dependencies.
I'll come up with a separate PR for the same.